### PR TITLE
More exercises

### PR DESCRIPTION
**Pull request by kariabancroft**
## Description
Rename pyramid exercise, add a new DateSelector exercise. DateSelector is not good for intro OO but is good for additional practice in using built-in Ruby functionality and/or reading the docs.

## Todos
None

## Feedback Requested
This PR also goes along with [this PR](https://github.com/Ada-Developers-Academy/Teachers-Edition-Textbook/pull/10) from the TET with a potential solution/option for the DateSelector.

### CODE REVIEW

File: 01-ruby-fundamentals/exercises/date-selector-class.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+The class should define the following instance methods:
+- A `start` method that returns the date of the first day of the period.
+  - A day starts on the day given.
+  - A week starts on the Monday that is or is just before the date.
+  - The month is the first day of the month.
+  - The year does not change.
+- An `end` method that returns the date of the last day of the period.
+  - A day ends on the day given.
+  - A week ends on the Sunday that is or is just after the date.
+  - Though not always true, you can assume a month has 31 days.
```
Comment (droberts-ada) [inline 01-ruby-fundamentals/exercises/date-selector-class.md]: Maybe add an optional to make the month end on the correct day?

File: 01-ruby-fundamentals/exercises/date-selector-class.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  - The year does not change.
+- Attributes `date` and `period`.
+- A `range_text` method that returns a string for the range in the following formats:
+    - For a day: “11/19/2015”
+    - For a week: “Week ending 11/22/2015”
```
Comment (droberts-ada) [inline 01-ruby-fundamentals/exercises/date-selector-class.md]: "Week beginning <date>" is how I usually think about weeks - did you chose "week ending" for a reason?

Comment (kariabancroft) [inline 01-ruby-fundamentals/exercises/date-selector-class.md]: nope! :-D

**Comment by droberts-ada** [comment]
[More exercises](https://trello.com/c/lElX9sqR/344-more-exercises)

**Pull request merged**